gemfirexd/tools/src/dunit/java/com/pivotal/gemfirexd/ClientServerDUnit.java

```diff
@@ -1749,8 +1749,8 @@ public void testNetworkClientFailover() throws Exception {
     // keep one statement prepared with args to check if it works fine
     // after failover
     PreparedStatement pstmt31 = conn3
-        .prepareStatement("select * from sys.members where kind <> ?");
-    pstmt31.setString(1, "locator(normal)");
+        .prepareStatement("select * from sys.members where kind = ?");
+    pstmt31.setString(1, "datastore(normal)");
```

Comment:
I did not get why this change was necessary?

Reply:
Because there is also a client now.